src/handlers/install_handler.rs

```diff
@@ -29,34 +29,33 @@ pub async fn start(
     let is_version_installed =
         helpers::version::is_version_installed(&version.tag_name, config).await?;
 
-    if is_version_installed && version.version_type != VersionType::Nightly {
-        return Ok(InstallResult::VersionAlreadyInstalled);
-    }
-
-    let mut nightly_version = None;
-
-    if is_version_installed && version.version_type == VersionType::Nightly {
-
-        info!("Looking for nightly updates");
-
-        nightly_version = Some(helpers::version::nightly::get_upstream_nightly(client).await?);
-        let upstream_nightly = nightly_version.as_ref().unwrap();
-        let local_nightly = helpers::version::nightly::get_local_nightly(config).await?;
-
-        if upstream_nightly.published_at == local_nightly.published_at {
-            return Ok(InstallResult::NightlyIsUpdated);
-        }
+    let nightly_version = if version.version_type == VersionType::Nightly {
```

Comment:
Why are you reverting back to the old way I handled nightly? I changed it cuz it had too much indentation and looked pretty ugly

Reply:
Fixed